### ISSUE TITLE
JDK home may differ from JDK source location on *buntu distros

### ISSUE THREAD

**Issue by jsjolen** [ensime/ensime-sbt#405]
On Kubuntu 18.04 (and probably other *buntus) the OpenJDK11 install (package: openjdk-11-jdk) is put in "/usr/lib/jvm/java-11-openjdk-amd64/" and the OpenJDK11 source (package: openjdk-11-source) is put in "/usr/lib/jvm/openjdk-11/lib/src.zip".

The issue with this is that ensimeConfig looks for the sources only in the java home directory, as seen here: https://github.com/ensime/ensime-sbt/blob/2.0/src/main/scala/EnsimePlugin.scala#L391

It would be useful if there was a way of specifying the javaSrc folde, now I resort to manually editing the .ensime file afterwards.

**Comment by jsjolen** [comment]
It seems like there's an args argument that is parsed. Adding an option for the source there would be great. I've never written Scala before (hence why I'm setting up Ensime in preparation).

If I write a pull request adding an option (deetz below) is there a possibility that it'd be merged?

I was thinking:

```
sbt configEnsime --javaSrc path/to/src
```

**Comment by amanjpro** [comment]
> It seems like there's an args argument that is parsed. Adding an option for the source there would be great. I've never written Scala before (hence why I'm setting up Ensime in preparation).
> 
> If I write a pull request adding an option (deetz below) is there a possibility that it'd be merged?
> 
> I was thinking:
> 
> ```
> sbt configEnsime --javaSrc path/to/src
> ```

I believe it will be better to have the option set at the global setting level:

https://www.scala-sbt.org/1.0/docs/Global-Settings.html

**Comment by jsjolen** [comment]
I agree, but we would still have to add a 'ensimeJavaSrcHome`, right now the following will fail on running the JVM but will find the sources:

```
ensimeJavaHome in ThisBuild := file("/usr/lib/jvm/openjdk-11/lib/")
```
